Jenkinsfile

```diff
@@ -25,11 +25,12 @@ pipeline {
                             sh """
                             python3 -m venv env
                             source env/bin/activate
-                            pip install -r requirements.txt
```

Comment:
I liked the requirement-file, because everyone knows it holds the python-deps of the repo.

Just listing this in the Jenkinsfile is less clear and confusing.

Reply:
It is not strictly required. It's just a plugin Jenkins uses to run the tests in parallel.

Reply:
Good point. I addressed it in a commit.